lib/main.dart

```diff
@@ -141,10 +141,15 @@ void main() async {
   //Update language header in interceptor on language change.
   appSettingsCubit.stream.listen((event) => languageHeaderInterceptor
       .preferredLocaleSubtag = event.preferredLocaleSubtag);
+  
+try {
```

Comment:
I'd prefer to use platform guards instead of try catch like this:
```dart
if (!Platform.isIOS) {
  await FlutterDisplayMode.setHighRefreshRate();
}
```

Or do you think this is not just IOS related?

Reply:
Could you check the recent changes on the development branch? I've added another approach to setting the refresh rate, could you possibly check if this also works on iOS?

Reply:
Platform guards was also my first approach 😄  Had changed it to try/catch because of a probably problem for older android devices too. Anyways, I have change it back and on developer branch it works with the new approach by it own.

Reply:
Hmm I have run the development branch inside the iOS Simulator and here I run also into a unhandled exception

> [VERBOSE-2:dart_vm_initializer.cc(41)] Unhandled Exception: MissingPluginException(No implementation found for method getSupportedModes on channel flutter_display_mode)

So I would say `flutter_displaymode` don't implement all possible targets and it is a good practice to try/catch here.

Reply:
Alright, thanks for trying it out anyway :) I just thought it would be a cleaner solution, try catch is fine then.

Reply:
I looked at the package, and it is only available for android anyway. So I think we can put a if (Platform.isAndroid) guard around it.